src/NuGet.Clients/NuGet.VisualStudio.Common/ProjectHelper.cs

```diff
@@ -5,9 +5,7 @@
 using EnvDTE;
 using Microsoft.VisualStudio;
 using Microsoft.VisualStudio.ProjectSystem;
-#if VS14
-using Microsoft.VisualStudio.ProjectSystem.Designers;
-#elif VS15
+#if VS15
 using Microsoft.VisualStudio.ProjectSystem.Properties;
 #endif
```

Comment:
Doesn't this #endif need removed?

Reply:
we still have a `#if VS15` here, should we remove that one as well?